nexus/utils/index.ts

```diff
@@ -152,30 +154,39 @@ export const readTime = (text: string): number => {
 export const updatedThreadPositions = (
   oldDoc: NodeType[],
   newDoc: NodeType[],
-  threads: Thread[]
-) => {
+  threads: Thread[],
+): {
+  startIndex: number
+  endIndex: number
+  id: number
+  archived: boolean
+  highlightedContent: string
+  postId: number
+}[] => {
```

Comment:
Didn't think about it! Thank you for reviewing it. Just sent another commit using types from the project.

> So we try to only explicitly add types when the TS compiler cannot figure it out.

Totally agree, but I did it because saw a bunch of warnings landing at the file, as I was there, just fixed it.

![Screen Shot 2021-01-08 at 19 51 58](https://user-images.githubusercontent.com/9276511/104073160-b9205380-51eb-11eb-9f64-4eb84e47e6b8.png)

I usually remove this kind of warning on my projects that I'm working on with `"@typescript-eslint/explicit-function-return-type": "off",`. If it sounds good I can create another PR to discuss more it.

Reply:
Oh that's interesting, I don't get the explicit return type warning so maybe I turned it off in my local configuration. I think creating an issue for that separately sound good!